a.js

```diff
@@ -97,15 +97,16 @@ await page.keyboard.press("Enter");
   await page.waitForTimeout(1000);
   await page.keyboard.press("Enter");
 
-  await page.waitForSelector(".VfPpkd-LgbsSe-OWXEXe-k8QpJ", { visible: true, timeout: 30000 });
-  await page.click(".VfPpkd-LgbsSe-OWXEXe-k8QpJ");
-  await page.waitForTimeout(1000);
-  await page.keyboard.press("Enter");
+  await page.waitForSelector('[aria-label="Search"]', { visible: true, timeout: 30000 });
```

Comment:
the url to scrape flights should be `https://www.google.com/travel/flights?q=Flights%20to%20{dest}%20from%20{org}%20on%20{dl}%20through%20{dr}` just like the `scrape.py `file